perf/specjbb/playlist.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+    <include>run/testenv.mk</include>
+    <test>
+        <!-- The minimum hardware requirement to run this test is 8GB of memory and 8 CPU cores -->
+        <!-- see specjbb/run/options/specjbb-multijvm-simple.sh for more details -->
+
+        <testCaseName>SPECjbb2015-multijvm-simple</testCaseName>
+        <command>. $(RUN_OPTIONS)/specjbb-multijvm-simple.sh; bash $(RUN_SCRIPTS)/run_multi.sh; $(TEST_STATUS)</command>
+        <levels>
+            <level>extended</level>
+        </levels>
+        <groups>
+            <group>perf</group>
+        </groups>
```

Comment:
```suggestion
        </groups>
        <vendors>
            <vendor>microsoft</vendor>
         </vendors>
```

Reply:
This will ensure it only gets run for the listed vendors.  Once this is in, other vendors may want to add this to their suites by adding another `<vendor>` entry.  If all vendors wanted to run it, and only we at Adoptium could not, we could instead add a `<disables><disable><comment>Requires SPECjbb 2015 License</comment><vendor>eclipse</vendor></disable></disables>`

Reply:
Should I leave this suggested change out since I've already the other suggested change that added the `<disables>` tag: https://github.com/ahmedmuhsin/aqa-tests/blob/specjbb-initial-support/perf/specjbb/playlist.xml#L22 ?

Reply:
I think we should **not** use the `<disables><disable><comment>Requires SPECjbb 2015 License</comment><vendor>eclipse</vendor></disable></disables>` approach for now (which says everyone except Eclipse runs this test).  I mentioned we could move to that approach if we find that all other vendors (which include IBM, Alibaba, Huawei, Azul, Red Hat) would want to run this test.  Since we do not know that answer yet, we would only use `<vendors><vendor>microsoft</vendor></vendors>` approach (which says MSFT runs this test).

Reply:
@smlambert out of interest where does this value get set? Trying to work out how we (msft) would enable this test in our CI system

Reply:
its [auto-detected by TKG](https://github.com/adoptium/TKG/blob/master/src/org/openj9/envInfo/JavaInfo.java#L192), you won't have to change anything to enable this...